crates/bevy_window/src/window.rs

```diff
@@ -126,16 +126,13 @@ pub struct Window {
     /// Note: This does not stop the program from fullscreening/setting
     /// the size programmatically.
     pub resizable: bool,
-    /// Enables and disables window control buttons.
+    /// Specifies which window control buttons should be enabled.
     ///
     /// ## Platform-specific
     ///
     /// **`iOS`**, **`Android`**, and the **`Web`** do not have window control buttons.
     ///
-    /// macOS note: When `resizable` is set to `false` the maximize button will be
-    /// disabled regardless of the value of the `EnabledButtons` struct. Additionaly, when
-    /// `resizable` is set to `true` the window will be maximized when its bar is double-clicked
-    /// regardless of whether the maximize button is enabled or not.
+    /// On some **`Linux`** environments these values have no effect.
```

Comment:
I think this line is not really necessary here, it's already in `EnableButtons`, but idk.

Reply:
I put it there as I think it goes along the iOS/Android/Web note as they are all Platform-specific notes. I put a newline inbetween as IntelliSense was showing both sentences in one line if they were one beneath the other.

Reply:
Ah, got it!

Reply:
Maybe the whole "Platform Specific" section shouldn't be here?
It is here for `resizable` because it's a boolean, but for struct people can just go on the struct page.

I'm fine with both, just opening a conversation.

Reply:
`WindowLevel` has the "Platform Specific" section on both the struct and the `Window` struct member. The `window_theme` member has the section as well even tho it's not present on the `WindowTheme` struct, however this struct is contained within an `Option`.

Reply:
Okay then ^^